.gitignore

```diff
@@ -19,3 +19,6 @@
 /public/packs-test
 /yarn-error.log
 yarn-debug.log*
+
+.idea
+.DS_Store
```

Comment:
please add .idea and .DS_Store to your machine-wide gitignore. don't put your personal editor or OS decisions into a shared repo. 😄  see https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/ for more discussion

Reply:
fixed